py3-cython.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: py3-cython
+  version: 3.0.10
+  epoch: 1
+  description: Cython is an optimising static compiler for both the Python & the extended Cython programming languages.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provider-priority: 0
+
+vars:
+  pypi-package: cython
+
+data:
+  - name: py-versions
+    items:
+      3.10: "310"
+      3.11: "311"
+      3.12: "312"
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
```

Comment:
fwiw, build-base has a dependency on wolfi-baselayout, which has a dependency on the ca-certificates-bundle, so we don't really need this.
https://github.com/wolfi-dev/os/blob/d801266df70d33f3f8aba620110f17182621e4e7/build-base.yaml#L15

https://github.com/wolfi-dev/os/blob/main/wolfi-baselayout.yaml#L10

Since you're cleaning this, just a heads up that it can be removed from all of these.